src/rgw/rgw_log.cc

```diff
@@ -346,7 +346,11 @@ int rgw_log_op(RGWRados *store, RGWREST* const rest, struct req_state *s,
   else
     set_param_str(s, "REMOTE_ADDR", entry.remote_addr);
   set_param_str(s, "HTTP_USER_AGENT", entry.user_agent);
-  set_param_str(s, "HTTP_REFERRER", entry.referrer);
```

Comment:
As AWS S3 are using `referrer` (see http://docs.aws.amazon.com/AmazonS3/latest/dev/LogFormat.html Referrer Section), rgw are correctly following this convention, but curl -e option are still using misspelling `referer` . I think it's a problem of legacy client, this purpose of this pr is to improve the rgw compatibility.